.github/workflows/automation-prod.yml

```diff
@@ -18,4 +18,3 @@ jobs:
         uses: cypress-io/github-action@v4
         with:
           env: youth_pass_url=https://mbta.prod.simpligov.com/prod/portal/ShowWorkFlow/AnonymousEmbed/e5e2dba6-5424-48d6-80d9-16a27e0fdd84,senior_url=https://mbta.prod.simpligov.com/prod/portal/ShowWorkFlow/AnonymousEmbed/fc6ff5b0-f3bd-436d-8201-b6a4b98bc7fa
```

Comment:
do we want to remove the `youth_pass_url` here too?

Reply:
no. youth pass is still tested. the tests just assert that the retirement page is shown and the "submit" button is hidden